sakai-evaluation-tool/src/java/org/sakaiproject/evaluation/tool/reporting/XLSReportExporter.java

```diff
@@ -79,223 +95,627 @@ public void setMessageLocator(MessageLocator locator) {
+     * 
+     * @see
+     * org.sakaiproject.evaluation.tool.reporting.ReportExporter#buildReport(org.sakaiproject.evaluation
+     * .model.EvalEvaluation, java.lang.String[], java.lang.String, java.io.OutputStream)
+     */
+    public void buildReport(EvalEvaluation evaluation, String[] groupIds, String evaluateeId, OutputStream outputStream, boolean newReportStyle) {
+    	
+        /*
+         * Logic for creating this view 1) make tidl 2) get DTIs for this eval from tidl 3) use DTIs
+         * to make the headers 4) get responseIds from tidl 5) loop over response ids 6) loop over
+         * DTIs 7) check answersmap for an answer, if there put in cell, if missing, insert blank 8)
+         * done
+         */
+
+    	
+    	//Make sure responseAggregator is using this messageLocator
+        responseAggregator.setMessageLocator(messageLocator);
+
+        // Determine which report style to use; normal or section based
+        if( newReportStyle )
+        {
+            buildReportSectionAware( evaluation, groupIds, outputStream );
+        }
+        else
+        {
+            Boolean instructorViewAllResults = (boolean) evaluation.getInstructorViewAllResults();
+            String currentUserId = commonLogic.getCurrentUserId();
+            String evalOwner = evaluation.getOwner();
+
+            boolean isCurrentUserAdmin = commonLogic.isUserAdmin(currentUserId);
+
+           // 1 Make TIDL
+           TemplateItemDataList tidl = getEvalTIDL(evaluation, groupIds);
+           // 2: get DTIs for this eval from tidl
+           List<DataTemplateItem> dtiList = tidl.getFlatListOfDataTemplateItems(true);
+
+           Workbook wb;
+           if( dtiList.size() < 256 )
+           {
+               wb = new HSSFWorkbook();
+           }
+           else
+           {
+               wb = new XSSFWorkbook();
+           }
+           creationHelper = wb.getCreationHelper();
+
+           Sheet sheet = wb.createSheet(messageLocator.getMessage("reporting.xls.sheetname"));
+
+           // Title Style
+           Font font = wb.createFont();
+           font.setFontHeightInPoints((short) 12);
+           font.setBoldweight(Font.BOLDWEIGHT_BOLD);
+           CellStyle mainTitleStyle = wb.createCellStyle();
+           mainTitleStyle.setFont(font);
+
+           // Bold header style
+           font = wb.createFont();
+           font.setFontHeightInPoints((short) 10);
+           font.setBoldweight(Font.BOLDWEIGHT_BOLD);
+           CellStyle boldHeaderStyle = wb.createCellStyle();
+           boldHeaderStyle.setFont(font);
+
+           // Italic meta header style
+           font = wb.createFont();
+           font.setFontHeightInPoints((short) 10);
+           font.setItalic(true);
+           CellStyle italicMiniHeaderStyle = wb.createCellStyle();
+           italicMiniHeaderStyle.setFont(font);
+
+           // Date meta Style
+           dateCellStyle = wb.createCellStyle();
+           // TODO FIXME HELPME To properly
```

Comment:
This is old code that I didn't actually write. Not sure why it's showing up in the diff, probably line endings or whitespace changes.